<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MCDVisualizerDockWidgetBase</class>
 <widget class="QDockWidget" name="MCDVisualizerDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>941</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>396</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>396</width>
    <height>950</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="windowTitle">
   <string>MCD_Visualizer</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QLabel" name="Label_Epoca">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Year Time</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Epoca">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Yearly Average</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cold</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Warm</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stormy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 24 </string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 27</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 28</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 29</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 30</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 31</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 33</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 34</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martian Year 35</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Combo_Archivo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Hora_Local">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Local Time</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Hora">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>12</number>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Altitud">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Altitude (km)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Altitud">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Label_Latitud">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Latitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Longitud">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>340</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Longitude</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Longitud_Min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Min</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Label_Variable">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>490</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Select Variables to Display</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Push_Visualizar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>860</y>
      <width>211</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Check_Mapa">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>351</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Show full map (Ignore Lat/Lon)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Combo_Latitud_Min">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Min</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Combo_Latitud_Max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>260</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Max</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="Combo_Longitud_Max">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>370</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Max</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Label_Archivo">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Push_Reset">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Combo_Variable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>371</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Push_InterpConfig">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string>INTERPOLATION CONFIGURATION</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Spin_Latitud_Min">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Spin_Latitud_Max">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Spin_Longitud_Min">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="Spin_Longitud_Max">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
